starknet_py/utils/merkle_tree_test.py

```diff
@@ -0,0 +1,142 @@
+from typing import List
+
+import pytest
+from poseidon_py.poseidon_hash import poseidon_hash
+
+from starknet_py.hash.hash_method import HashMethod
+from starknet_py.hash.utils import pedersen_hash
+from starknet_py.utils.merkle_tree import MerkleTree
+
+
+@pytest.mark.parametrize(
```

Comment:
Just to confirm - have the examples below been taken from other SDKs?

Reply:
Yup, they've been taken from [Swift SDK](https://github.com/software-mansion/starknet.swift/pull/167/files#diff-38e25e09052ab3aafadd4ca0ab04af05e2737084545cf2a61c2bd1075dacaf78).